plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityKnockback.java

```diff
@@ -49,7 +55,14 @@ public static void register() {
         // Returns the knockback strength of an arrow or trident.

```

Comment:
The meta for these should probably have the deprecation as well? (can see other deprecated meta for format reference)